src/core.rs

```diff
@@ -108,6 +108,18 @@ pub struct Cookie {
     raw: Vec<u8>
 }
 
+impl Cookie {
+    pub fn into_bytes(self) -> Vec<u8> {
+        self.raw
+    }
+
+    pub fn from_bytes(bytes: &[u8]) -> Cookie {
```

Comment:
Why?

Reply:
Deserialization is defined in `ws-tox-protocol`, and it needs to construct a Cookie somehow.

Reply:
You could use `From<Vec<u8>> for Cookie`, no?